README.md

```diff
@@ -58,29 +58,29 @@ While other versions often work, the above are what we officially support. YMMV!
 
 We are active and enthusiastic participants of the following community support channels:
 
-- [TezosTaquito Telegram Community Channel][telegram]
+- [ECAD Labs Discord Channel][discord]
 - [Tezos StackExchange][stackexchange]
 
 ## Project Organization
 
-`taquito` is organized as a [mono repo](https://en.wikipedia.org/wiki/Monorepo), and includes several npm packages published to npmjs.org under the `@taquito` handle. Each package has its own README and can be found in the `packages/` directory
+Taquito is organized as a [monorepo](https://en.wikipedia.org/wiki/Monorepo), and is compose of several npm packages that are [published to npmjs.org](https://www.npmjs.com/package/@taquito/taquito) under the `@taquito` handle. Each package has its own README which can be found in the corresponding directory within `packages/`.
```

Comment:
compose -> composed

Reply:
Good catch! :+1: 
I will *not* be correcting your French anytime soon :joy:.